app/controllers/concerns/idv/passport_concern.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+module Idv
+  module PassportConcern
+    extend ActiveSupport::Concern
+
+    def doc_auth_passports_enabled?
+      IdentityConfig.store.doc_auth_passports_enabled
+    end
```

Comment:
Is it expected that there will be other conditions? If not, would it be preferable to just call the configuration directly?